.github/workflows/docker-build.yml

```diff
@@ -59,9 +60,9 @@ jobs:
           FLEXFLOW_CONTAINER_TOKEN: ${{ secrets.FLEXFLOW_CONTAINER_TOKEN }}
           FF_GPU_BACKEND: ${{ matrix.gpu_backend }}
         run: |
-          if [[ ${{ github.event_name }} == 'push' && ${GITHUB_REF#refs/heads/} == "master" ]]; then
-            ./docker/publish.sh "flexflow-environment-${FF_GPU_BACKEND}"
-            ./docker/publish.sh "flexflow-${FF_GPU_BACKEND}"
+          if [[ ${{ github.event_name }} == 'push' && (${GITHUB_REF#refs/heads/} == "master" || ${GITHUB_REF#refs/heads/} == "inference") ]]; then
+            # ./docker/publish.sh "flexflow-environment-${FF_GPU_BACKEND}"
```

Comment:
Here, I think we should still publish the `flexflow-environment` image when the workflow is triggered by the master branch, but we don't need to do so for the inference branch. The code right now never publishes it. How about we add another if statement to check the branch name and decide whether to publish the `flexflow-environment` based on that?